### ISSUE TITLE
Animations don't show in version >= 2.7.0

### ISSUE THREAD

**Issue by stevehurcombe** [martijn00/LottieXamarin#213]
## 🐛 Bug Report

I'm testing a simple animation, just to get started.

```
<lottie:AnimationView 
  HorizontalOptions="FillAndExpand" 
  VerticalOptions="FillAndExpand"
  Animation="4930-checkbox-animation.json" 
  HardwareAcceleration="False"
  Loop="True"
  AutoPlay="True" />
```
Nothing displays at all. I did think it was the HardwareAcceleration, but setting this to False had no effect.

Version 2.6.3 is fine.

There are no errors or exceptions.

### Expected behavior
Animation displays

### Configuration
Samsung Galaxy S7 Edge. 
XF 4 pre-5
**Version:** 2.7.x

**Platform:** 
Android

**Comment by vincentcastagna** [comment]
It does not show because you set Horizontal & VerticalOptions to "FillAndExpand". If you set a Width and HeightRequest value it will display, but we agree that, then, animation is not responsive anymore.

**Comment by stevehurcombe** [comment]
Thanks. 

Are they needed in 2.7.0? It worked exactly as shown in 2.6.3.

**Comment by vincentcastagna** [comment]
I had to do it also in 2.7.0 ...

**Comment by stevehurcombe** [comment]
This didn't work (2.7.1):

```
<lottie:AnimationView 
  HeightRequest="50" 
  WidthRequest="50"
  Animation="4930-checkbox-animation.json" 
  HardwareAcceleration="False"
  Loop="True"
  AutoPlay="True" />
```

**Comment by vincentcastagna** [comment]
Try this : 
```
<Grid>
<lottie:AnimationView 
  HeightRequest="50" 
  WidthRequest="50"
  Animation="4930-checkbox-animation.json" 
  HardwareAcceleration="False"
  Loop="True"
  AutoPlay="True" />
</Grid>
```

**Comment by canmrt** [comment]
> Try this :
> 
> ```
> <Grid>
> <lottie:AnimationView 
>   HeightRequest="50" 
>   WidthRequest="50"
>   Animation="4930-checkbox-animation.json" 
>   HardwareAcceleration="False"
>   Loop="True"
>   AutoPlay="True" />
> </Grid>
> ```

I had the same issue, animations were working on iOS simulator **XS Max** but it was not working on **8 Plus**. Both with iOS 12.1. This worked for me, but I don't understand why adding a grid solves the problem?

On Android, I haven't tried yet.

**Comment by caseyryan** [comment]
I'm facing the same issue. I have installed Lottie Forms 2.7.0 to my Xamarin Forms project but I can't make animations play.
There is no AnimationViewRenderer in lottie's android library and no way to use it. Only the iOS version of the renderer is available. If I downgrade the version to 2.5.2.1 the renderer is there and after calling Init() on it, all animations work fine. I tried to reinstall the library twice with the same result. Only the older versions work.
What's wrong with the 2.7.0 version?

**Comment by stevehurcombe** [comment]
Wrapping it in a Grid didn't work. I did set a background colour and it is being allocated space on the form.

**Comment by vincentcastagna** [comment]
@stevehurcombe  That's strange. I'm on 2.7.1 and no prob at all with a lottie with fixed height & width inside a grid. Is your lottie (wrapped or not in a grid) in another layout ? I noticed there are different behaviors, depending how your animation is wrapped in layouts.

**Comment by caseyryan** [comment]
Grid wrapping didn't work for me either

**Comment by vincentcastagna** [comment]
@caseyryan Try 2.7.1? Plz share your xaml where you display your lottie animation. 

Here is a working example in my app, with lottie 2.7.1, XF 3.4.0.1029999/ Prism 7.1 & Xamarin.Android libraries 27.0.2.1. 

```
<ScrollView VerticalOptions="FillAndExpand">
                <Grid>
                    <Image Opacity="1"
                       Source="main_background_image"
                       Aspect="AspectFill" VerticalOptions="FillAndExpand" />
                    <Grid  IsVisible="{Binding IsBusy}" BackgroundColor="#801A2634">
                        <lottie:AnimationView x:Name="busyindicator" HorizontalOptions="Center" VerticalOptions="Center" Speed="3" WidthRequest="160" Loop="True" AutoPlay="True" HeightRequest="90" Animation="loader2.json">
                        </lottie:AnimationView>
                        <Label Margin="0,100,0,0" Text="Chargement..." HorizontalOptions="Center" VerticalOptions="Center" TextColor="{StaticResource PrimaryColorThree}"></Label>
                    </Grid>
                    <partialviews:PartialViewCarousel IsVisible="{Binding IsBusy, Converter={StaticResource NegateBooleanConverter}}" Margin="0,10,0,0" mvvm:ViewModelLocator.AutowirePartialView="{x:Reference MainPage}" />
                </Grid>
            </ScrollView>
```

**Comment by caseyryan** [comment]
I'm doing it like this:

```
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             BackgroundColor="White"
             xmlns:lottie="clr-namespace:Lottie.Forms;assembly=Lottie.Forms"
             x:Class="MaxPlayer.Views.SplashPage">
    <ContentPage.Content>
        <Grid x:Name="PageOuterGrid">
            <lottie:AnimationView 
                x:Name="LoadingAnimation"
                Loop="True"
                AutoPlay="True"
                HeightRequest="500"
                WidthRequest="500"
                Animation="loading_anim.json"
                />

            
        </Grid>
    </ContentPage.Content>
</ContentPage>
```

This code works with the older versions but doesn't do anything in 2.7.0

I'll give a try to 2.7.1 (I hadn't known it's been out until today) and report about the results :)
p.s. I'm using Xamarin 4 (but not with experimental api) and Android SDK 28

**Comment by caseyryan** [comment]
I've just checked version 2.7.1. It's ok with this version. There is an  AnimationViewRenderer in this version so I can initialize it :) It works!

**Comment by mabdollahiasl** [comment]
> I've just checked version 2.7.1. It's ok with this version. There is an  AnimationViewRenderer in this version so I can initialize it :) It works!

I have this problem too, can you explain what did you do?
I cannot find this "AnimationViewRenderer "

**Comment by galadril** [comment]
I have the same issue.. can't update from 2.6.3 or animations are not showing up.

```

 <AbsoluteLayout HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand">
         <!--  Data  -->
         <view:DataView AbsoluteLayout.LayoutBounds="0,0,1,1" AbsoluteLayout.LayoutFlags="All" />

         <!--  Loading indicator  -->
         <lottie:AnimationView
            Margin="5"
            AbsoluteLayout.LayoutBounds="0.5,0.5,AutoSize,AutoSize"
            AbsoluteLayout.LayoutFlags="PositionProportional"
            IsVisible="{Binding Status}" />
</AbsoluteLayout>

```

And set the next parameters via resource files:

```

 <!--  Default LoadingIndicator Style  -->
   <Style TargetType="lottie:AnimationView">
      <Setter Property="Animation" Value="ship.json" />
      <Setter Property="HeightRequest" Value="50" />
      <Setter Property="WidthRequest" Value="50" />
      <Setter Property="AutoPlay" Value="True" />
      <Setter Property="Loop" Value="True" />
      <Setter Property="HorizontalOptions" Value="Center" />
      <Setter Property="VerticalOptions" Value="Center" />
   </Style>

```

And this is working perfectly on **version 2.6.3** , but the animation doesn't show if I update.

Any ideas?

**Comment by caseyryan** [comment]
Just upgrade to version 2.7.1. It works perfectly. 2.7.0 didn't work

**Comment by stevehurcombe** [comment]
2.7.1 didn't work for me. I'm still using 2.6.3.

**Comment by galadril** [comment]
2.7.1 isn't the fix for me either

**Comment by Ickerday** [comment]
I'm also unable to call AnimationViewRenderer.Init() in Android main activity and the animations don't work on that platform.

**Comment by flaviosuardi** [comment]
### Configuration
Huawei P20 lite
XF 4.0.0.425677
Xamarin.Android.Support.* 28.0.0.1
Platform: Android

### Issue
I have this `AnimationView` inside a `ContentView`:
```
var loaderAnimationView = new AnimationView
            {
                Animation = "Animations/loader_indefinite.json",
                Loop = true,
                AutoPlay = true,
                Margin = new Thickness(UIConstants.Padding16, 0, 0, 0),
                HorizontalOptions = LayoutOptions.FillAndExpand,
                VerticalOptions = LayoutOptions.FillAndExpand,
                HardwareAcceleration = false,
            };
```

Using 2.6.3 I have following error when I try to open the page contains previous object:
```
java.lang.IllegalStateException: Missing values for keyframe.
```

Using 2.7.1 animation doesn't show.

What can I do? I try everythings:
- use a original Lottie json file
- use older versions than 2.6*
- wrap `loaderAnimationView` in a `Grid`

but nothings seems work on Android (iOS is ok)

**Comment by Sedubald** [comment]
> I'm facing the same issue. I have installed Lottie Forms 2.7.0 to my Xamarin Forms project but I can't make animations play.
> There is no AnimationViewRenderer in lottie's android library and no way to use it. Only the iOS version of the renderer is available. If I downgrade the version to 2.5.2.1 the renderer is there and after calling Init() on it, all animations work fine. I tried to reinstall the library twice with the same result. Only the older versions work.
> What's wrong with the 2.7.0 version?

It is enough to go down to 2.6.x

**Comment by AbuMandour** [comment]
am facing the same issue with 
Xamarin.Forms.Lottie  3.0 
Xamarin.Forms 4.0.0.540
Microsoft Visual Studio Enterprise 2019
Version 16.0.3
Xamarin.Android SDK   9.2.3.0 
after downgrading to Xamarin.Forms.Lottie  2.6.3 it's working fine

**Comment by NirmalSubedi17** [comment]
Facing same issue. 
Lottie : > 3.0
Forms : > 4.0

Cann't downgrade, says Key frame missing :(

**Comment by UnreachableCode** [comment]
My animation shows from Xaml with LayoutOptions set to Fill. But I set the VerticalOptions to Center and suddenly the animation disappears.

**Comment by UnreachableCode** [comment]
This has now been fixed since upgrading from 3.0.1 to 3.0.2

**Comment by gionapaolini** [comment]
Still have the same problem with version 3.1.2
